src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -15,6 +15,14 @@ import { EndpointsService } from '../../../services/endpoints.service';
   styleUrls: ['./challengesubmit.component.scss']
 })
 export class ChallengesubmitComponent implements OnInit {
+ /**
+  * Url error Message
+  */
+inputErrorMessage = '';
+urlboxchecked = false;
+fileboxchecked = false;
+validFileUrl = false;
+onSubmissionOption: boolean;
```

Comment:
@Suryansh5545 Please check the flow once if it's working correctly cause there is a issue like when I tried submitting without uploading file so instead of showing error (`Please upload file`), it got submitted. Please check if all the above (^) variables are required and please check the functionality once before pushing.

Reply:
its not possible as you require to have file uploaded to click submit button

Reply:
yeah but now it is modified by you, please check once.

Reply:
i cant understand how it is possible to submit without uploading as the submit button is will nto enable but it have corrected some errors so it might be ok now

Reply:
Try submitting with url first then try switch back to file upload submission option then see... :)

Reply:
ok sir now i understand where i am missing .

Reply:
I am sorry for my mistake

Reply:
i have fixed the issue completely and now it is showing error when  trying it with file or URL field. but i have a suggestion when we show error should i disable the button again?

Reply:
sir i have put it so you can review it and decide

Reply:
no I think, it may confuse user if he forgot to upload file and he finds disabled button.

Reply:
so should i remove the whole submit button disabled thing? except that phase select

Reply:
so the formsubmit can provide the user with info need

Reply:
I think we should let it be like it was before :)

Reply:
by the way sir the button is disabled in previous code because the whole form was show and if user didnt select the phase and now the phase must to selected before submission . so this eliminates the need for one so should i remove the disable functionality?

Reply:
oh I see... then I think we can remove the disable functionality.

Reply:
ok sir